rest-api/offline/metrics_export.py

```diff
@@ -58,7 +59,8 @@
    AND NOT EXISTS
     (SELECT * FROM participant_summary ps
       WHERE ps.participant_id = ph.participant_id
-        AND ps.email LIKE :test_email_pattern)
+      AND ps.withdrawal_status != 2
```

Comment:
This should be instead:

ps.participant_id = ph.participant_id AND 
(ps.withdrawal_status = 2 OR ps.email LIKE :test_email_pattern)

because we're doing a NOT EXISTS here.